rset.go

```diff
@@ -513,11 +516,13 @@ func (rset *Rset) open(stmt *Stmt, ocistmt *C.OCIStmt) error {
 		if err != nil {
 			return err
 		}
+		cName := C.GoStringN(columnName, C.int(colSize))
 		Columns[n] = Column{
-			Name:   C.GoStringN(columnName, C.int(colSize)),
+			Name:   cName,
 			Type:   params[n].typeCode,
 			Length: params[n].columnSize,
 		}
+		ColumnIndex[cName] = n
```

Comment:
What about the duplicat column names?
Like `SELECT 1 a, 2 a FROM DUAL` ?

Reply:
Helper function it is.
Did not really get what you mean by "duplicate column names" :(

Reply:
Done the helper function. Can you merge now, please?